self-core-impl/src/test/java/com/selfxdsd/core/managers/StoredProjectManagerTestCase.java

```diff
@@ -125,6 +126,140 @@ public void returnsProjects() {
         );
     }
 
+    /**
+     * PmUser returns its username.
+     */
+    @Test
+    public void pmUserReturnsUsername() {
+        final Projects assigned = Mockito.mock(Projects.class);
+        final Projects all = Mockito.mock(Projects.class);
+        Mockito.when(all.assignedTo(1)).thenReturn(assigned);
+
+        final Storage storage = Mockito.mock(Storage.class);
+        Mockito.when(storage.projects()).thenReturn(all);
+
+
+        final ProjectManager ProjectManager = new StoredProjectManager(1,
+                "123",
+                "zoeself",
+                Provider.Names.GITLAB,
+                "123token",
+                storage);
+
+        User PmUser = new StoredProjectManager(ProjectManager.id(),
+                ProjectManager.userId(),
+                ProjectManager.username(),
+                Provider.Names.GITLAB,
+                "123token",
+                storage).new PmUser(ProjectManager);
```

Comment:
@HoreaPetrila You don't need to instantiate the parent class when instantiating a child class. Just:

``new StoredProjectManager.PmUser(...)``